frontend/src/components/dashboard/aliases/AliasGenerationButton.module.scss

```diff
@@ -17,7 +17,7 @@
   list-style-type: none;
   min-width: $content-xs;
   // Overlap the stats with position: relative in the alias cards:
-  z-index: 2;
+  z-index: 4;
```

Comment:
Question (non-blocking): What prompted the change?

Reply:
In prod, if you click generate new mask, you'll see that the mask card drop down icon is overlapping with the generate new mask menu,

![image](https://github.com/mozilla/fx-private-relay/assets/59676643/3c8cfda8-1d64-4c88-8269-96f543d0ff3f)